### PR TITLE
Mitigate QS Vulnerability

### PR DESCRIPTION
**Pull request by TommyJ1994**
Mitigate QS vulnerability (6.4.0)  https://issues.jboss.org/browse/RAINCATCH-603

Verified manually that the vulnerability is mitigated using `snyk test`

**3rd Party Module Fix Status**
-  request  -  2.81.0
-  body-parser - 1.17.1
-  express - 4.15.2

**Feedhenry Components Fix Status**
-  fh-mbaas-client - 0.16.5 
-  fh-reporting-client - 0.5.5

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
-    "express": "4.14.0",
+    "express": "4.15.2",
     "fh-amqp-js": "0.7.1",
-    "fh-mbaas-client": "0.15.0",
-    "fh-reportingclient": "0.5.4",
-    "multer": "0.1.8",
-    "request": "2.74.0",
+    "fh-mbaas-client": "0.16.5",
+    "fh-reportingclient": "0.5.5",
+    "multer": "0.0.7",
```
Comment (wtrocki) [inline package.json]: Downgrade?

Comment (TommyJ1994) [inline package.json]: Ah, that's a mistake. Thought it was coming up from v0.0.3, so many version changes this morning on this PR to get build right..

**Comment by TommyJ1994** [comment]
Need to investigate which version of multer to use. multer > v1.0.0 will break the tests.

**Comment by TommyJ1994** [comment]
retest this please

**Comment by TommyJ1994** [comment]
@wtrocki Quick review? I'm not going to mitigate the vulnerability in the multer dependency. It will only cause scope creep. It only accounts for 5 of the 120+ paths caused by this QS vulnerability.

**Comment by wtrocki** [comment]
@TommyJ1994 Does that mean that we would still get vulnerability reports for multer?

**Comment by TommyJ1994** [comment]
@wtrocki Not on PR's and not when running NSP Check. It will only show up when running `snyk test` manually.

**Pull request merged**